docs/source/runningjobs/interactiveandbatch.md

```diff
@@ -3,72 +3,101 @@
+- **`N, --nodes=<minnodes[-maxnodes]>`**: specify the number of nodes
+- **`p, --partition=<partition-name>`**: specify a partition for the job to run on
 
-(interactive-mode-use-cases)=
-### Interactive Mode Use Cases
-- **Development and Testing**: Ideal for code development and testing.
-- **Short Tasks**: Best for tasks that require less time and immediate results.
+To see all options for `srun`, please refer to [srun manual] from Schedmd.
 
+### Examples Using `srun`
 
-## Batch Mode
-Batch mode enables users to write scripts that manage job execution, making it suitable for more complex or longer-running jobs.
+You can tailor your request to fit both the needs of the job and the partition limits if you're familiar with the available hardware and partitions on Discovery. 
 
-### Creating Batch Scripts
-A typical batch script includes directives for resource allocation, job names, and commands. Here is an example:
+To request one node and one task for 30 minutes with X11 forwarding on the short partition, type:
 
 :::{code} bash
-#!/bin/bash
-#SBATCH --job-name=my_job
-#SBATCH --nodes=1
-#SBATCH --ntasks=4
-#SBATCH --time=01:00:00
+srun --partition=short --nodes=1 --ntasks=1 --x11 --mem=2G --time=00:30:00 --pty /bin/bash
+:::
+
+To request one node, with 10 tasks and 2 CPUs per task (a total of 20 CPUs), 40 GB of memory, for one hour on the express partition, type:
 
-# Commands to execute
-module load python/3.8.1
+:::{code} bash
+srun --partition=short --nodes 1 --ntasks 10 --cpus-per-task 2 --pty --mem=40G --time=01:00:00 /bin/bash
+:::
 
-python my_python_script.py
+To request 2 nodes, each with 10 tasks per node and 2 CPUs per task (a total of 40 CPUs), 80 GB of memory, for one hour on the express partition, type:
 
+:::{code} bash
+srun --partition=short --nodes=2 --ntasks 10 --cpus-per-task 2 --pty --mem=80G --time=01:00:00 /bin/bash
 :::
 
-Save this script with a `.sh` extension, e.g., my_script.sh.
+To allocate a GPU node, you should specify the `gpu` partition and use the `–gres` option:
+
+:::{code} bash
+srun --partition=gpu --nodes=1 --ntasks=1 --gres=gpu:1 --mem=2Gb --time=01:00:00 --pty /bin/bash
+:::
 
-### Submitting Batch Jobs
-You can submit your batch script using the sbatch command.
+(using-sbatch)=
+## Batch Jobs: `sbatch` Command
+The `sbatch` command is used to submit a job script for passive execution. The script includes the `SBATCH` directives that control the job parameters (e.g., number of nodes, CPUs per task, job name). To submit the batch jobs, the following is run from the login node:
 
 :::{code} bash
-sbatch my_script.sh
+sbatch [options]  <script_file>
 :::
 
-### Monitoring Batch Jobs
-You can monitor the status of your batch job using the squeue command.
+An example sbatch script for a job utilizing 2 nodes and 16 tasks per node:
 
 :::{code} bash
-squeue -u username
+#!/bin/bash
+#SBATCH -J MyJob                            # Job name
+#SBATCH -N 1                                # Number of nodes
```

Comment:
`#SBATCH -N 2`

Reply:
fixed this and added a note about only using more than 1 node if the program is optimized for multi-nodal execution.